extern/agg24-svn/include/agg_math_stroke.h

```diff
@@ -391,7 +391,8 @@ namespace agg
         vc.remove_all();
 
         double cp = cross_product(v0.x, v0.y, v1.x, v1.y, v2.x, v2.y);
-        if(cp != 0 && (cp > 0) == (m_width > 0))
```

Comment:
What is the operator prescience in c++?  Would

```c++
if(cp != 0 && ((cp > 0) == (m_width > 0)))
```

also work?

Reply:
`cp` is non-zero on the other arches; it doesn't really have to do with precedence. The switch to two separate comparisons is so that `-eps < cp < eps` is considered 0.

Reply:
See [this comment](https://github.com/matplotlib/matplotlib/issues/7797#issuecomment-336683254) from the issue.